### PR TITLE
fix: Add outlines reverting PR

### PR DESCRIPTION
**Pull request by akshatj2209**
There was a PR that needed to be fixed due to it having a negative impact on the accessibility of the website. I tried to revert the changes and hope that the issue is fixed. Please review and merge

### CODE REVIEW

File: components/Headers/IndexHeader.js [hunk 1]
```diff
@@ -100,70 +100,65 @@ function classNames(...classes) {
...
-            <Popover.Group as="nav" className="hidden md:flex space-x-10">
-              <Popover className="relative">
+            <Popover.Group as='nav' className='hidden md:flex space-x-10'>
+              <Popover className='relative'>
                 {({ open }) => (
                   <>
                     <Popover.Button
                       className={classNames(
                         open ? "text-gray-900" : "text-black",
                         "group bg-white rounded-md inline-flex items-center text-base font-medium hover:text-gray-500 focus:outline-none "
```
Comment (Gupta-Anubhav12) [inline components/Headers/IndexHeader.js]: Hey @akshatj2209 you missed this, please remove outline none from here too

**Review by Gupta-Anubhav12 — CHANGES_REQUESTED** [review]
please remove outline from chapter link too

**Comment by Gupta-Anubhav12** [comment]
Hey @akshatj2209 
Thanks for this contribution, the audioplayer is missing a audio length seeker as in original designs, please fix that too before we merge

**Comment by akshatj2209** [comment]
The Progress Bar has been added @Gupta-Anubhav12 , Please review it and specify any other requirements.

**Comment by Gupta-Anubhav12** [comment]
Hey it works smooth, great 
merging now

**Review by Gupta-Anubhav12 — APPROVED** [review]

**Comment by Gupta-Anubhav12** [comment]
There are some imperfections but we can merge it for MVP and look at those afterwards ,
Thanks for contributing @akshatj2209

**Pull request merged**